Update Installation Guides (#3988)
sparkdesignsystem/spark-design-system: src/pages/installing-spark/react.mdx

import reactApp from '../../images/installing-spark/react/react-app.png';
import styleCheck from '../../images/installing-spark/react/style-check.png';
import buttonLoaded from '../../images/installing-spark/react/button-loaded.png';
import spinnergif from '../../images/installing-spark/html/html-spinner.gif';

# Installing Spark in a React Project

This guide walks you through installing and using Spark on an existing
React app.

For help with setting up the necessary development environment,
see the [React App Getting Started Guide](https://create-react-app.dev/docs/getting-started).

By the end of this guide, you’ll have Spark set up on your project and have access to all of our components.

-----------

Our <a href="https://github.com/sparkdesignsystem/spark-starter-react/" target="_blank">starter app examples</a> are available for reference:
- ['kitchen-sink' branch](https://github.com/sparkdesignsystem/spark-starter-react/tree/kitchen-sink) (Spark Installed with component examples.)

> - ['without-spark' branch](https://github.com/sparkdesignsystem/spark-starter-react/tree/kitchen-sink) (All of the prerequisites before Spark Installation.)

Reviewer: ```suggestion
- ['without-spark' branch](https://github.com/sparkdesignsystem/spark-starter-react/tree/without-spark) (All of the prerequisites before Spark Installation.)
```

## Install Spark Packages
Now we’ll use Sass to import all the Spark CSS.

> 1. Install the Spark React package. It contains all of the Spark React components and Sass styles needed to build with spark.

Reviewer 2: ```suggestion
1. Install the Spark React package. It contains all of the Spark React components and Sass styles needed to build with Spark.
```

>     alt="Screenshot of React App with Spark Spark Button"/>

Reviewer: ```suggestion
    alt="Screenshot of React App with Spark Button"/>
```
